html/changelogs/Swimstars.yml

```diff
@@ -0,0 +1,6 @@
+author: ChangeMe
+
+delete-after: True
+
+changes:
+  - rscadd: "The all new Swimstars plushies & lunchbox has been approved for station usage. Please find them in hte loadout."
```

Comment:
```suggestion
  - rscadd: "The all new Swimstars plushies & lunchbox has been approved for station usage. Please find them in the loadout."
```

Reply:
I should really double check my writing in the future. Fixed.